### DISCUSSION TITLE
Problem with "scrollToZoom" option and Inline plugin

### DISCUSSION THREAD

**Discussion by edubeckha** [igordanchenko/yet-another-react-lightbox#247] [Q&A] [answered]
Hi guys,

I'm trying the "scrollToZoom" option on my Lightbox component, but when I use it the entire page keeps getting scrolled as well. Is there a way to disable this behavior?

Thank you in advance!

**Comment by igordanchenko** [comment]
Do you mind providing a minimal repro? Feel free to fork the following sandbox - https://stackblitz.com/edit/yet-another-react-lightbox-sandbox?file=src%2FApp.tsx

**Reply by edubeckha** [reply]
Hi Igor, 

I created a minimal example of what I'm trying to achieve here: https://stackblitz.com/edit/yet-another-react-lightbox-sandbox-52ybef?file=src%2Fmain.tsx

It's easier to notice this behaviour if you open it in a new window. When I have other components (like a bunch of text for example) before and after the image gallery and try to apply zoom to the image only using the mouse scroll, the entire page scrolls up or down. 

It seems that maybe there is a problem with event propagation coming from the gallery to the window, but I'm not sure how to stop it from happenning.

Thank you again!

**Reply by igordanchenko** [reply, marked as answer]
Thank you for the repro, this is very helpful. Indeed, propagation of wheel events needs to be cancelled in this case. I'll address this in the next patch release.

**Reply by edubeckha** [reply]
Thanks Igor, I appreciate your help.

**Reply by igordanchenko** [reply]
This is now available in v3.17.1

**Reply by edubeckha** [reply]
Thank you Igor, it works really well now.
